### PR TITLE
Kyle

### PR DESCRIPTION
**Pull request by emeraldarcher**
Roll Controller finished

### CODE REVIEW

File: java/src/client/roll/RollController.java [hunk 1]
```diff
@@ -19,6 +21,7 @@
 	public RollController(IRollView view, IRollResultView resultView) {
 		super(view);
 		setResultView(resultView);
+		roller = new Dice(2,12);
```
Comment (CaptBoscho) [inline java/src/client/roll/RollController.java]: if you do it just like this, then the percentages of numbers rolled will be more equal than 2 die... like 2 has the same probability of being rolled as a 7. I would recommend making two die from 1-6 and adding up their rolls. What do you think?

File: java/src/client/turntracker/TurnTrackerController.java [hunk 2]
```diff
@@ -21,12 +33,30 @@ public ITurnTrackerView getView() {
 
 	@Override
 	public void endTurn() {
-
+//		facade.endTurn(); // TODO: 2/16/2016 Have Corbin implement in facade
```
Comment (CaptBoscho) [inline java/src/client/turntracker/TurnTrackerController.java]: It is finishTurn(int playerID). Requires the playerID just as a check.

File: java/src/client/turntracker/TurnTrackerControllerState.java [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+     */
+    public TurnTrackerControllerState(Game model){
+        gModel = model;
+        facade = Facade.getInstance();
+        currentPlayerID = facade.getCurrentTurn();
+        initFromModel();
+    }
+
+    public void endTurn() {
+//		facade.endTurn(); // TODO: 2/16/2016 Have Corbin implement in facade
```
Comment (CaptBoscho) [inline java/src/client/turntracker/TurnTrackerControllerState.java]: :)

**Comment by derekargueta** [comment]
Currently don't have a way to test this yet. Are there any other game components that depend on this being merged into master ASAP?

**Comment by emeraldarcher** [comment]
Is it possible to pick and choose what to merge? I have some changes outside of the controllers I'm working on that should be merged.

**Pull request merged**